doc/changelog/08-vernac-commands-and-options/16004-pr-Alizter-16004.rst

```diff
@@ -0,0 +1,13 @@
+- **Changed:**
+  `Hint` (and `Instance`) commands with no locality attribute are deprecated.
+  Previous versions would raise a warning, but now this has been changed to an
+  error. This includes:
+  - All `Hint` commands.
+  - `Hint Rewrite`
+  - `Instance`
```

Comment:
You could hyperlink, e.g. ``` :cmd:`Hint Rewrite` ``` for the 2 commands and a link to the section for All Hint commands